packages/bundle-source/src/index.js

```diff
@@ -17,8 +17,6 @@ import { encodeBase64 } from '@endo/base64';
 import SourceMaps from 'source-map';
 import { whereEndoCache } from '@endo/where';
 
```

Comment:
```suggestion

export * from './exports.js';
```

Reply:
I moved the guts of this file to `src/bundle-source.js`, leaving `src/index.js` to export the types and the implementation.